core/src/main/java/io/apicurio/hub/api/codegen/OpenApi2JaxRs.java

```diff
@@ -606,6 +629,12 @@ static Type<?> parseType(String value) {
         });
     }
 
+    AnnotationSource<?> parseParameterAnnotation(String annotation) {
+        String stub = "public class Stub { public void method( " + annotation + " Object arg0 ) {} }";
```

Comment:
this looks a bit of an hackish way to extract a new `AnnotationSource`, and I imagine that is also computationally pretty expensive ... no alternatives?

Reply:
I agree that this is a bit icky, but it does appear to be the modus operandi for Roaster. See [1]. I can dig around a bit more and see if there is a better alternative.

[1] https://github.com/forge/roaster/blob/1a16012a439200cfb6671c23472cea286b8d055a/impl/src/main/java/org/jboss/forge/roaster/model/impl/TypeImpl.java#L56-L68

Reply:
I'm not familiar with Roaster, but, from a quick look seems like you are right:
https://github.com/forge/roaster/blob/1a16012a439200cfb6671c23472cea286b8d055a/impl/src/main/java/org/jboss/forge/roaster/model/impl/AnnotationImpl.java#L332

I think it would be best to see if we can rely on some built-in functions from Roaster, since this might be a shortcoming that can be improved in the future; but, if you don't find anything better, I'm ok with this approach.

Reply:
I'm not seeing anything to indicate there is a way to create elements of a Java source file in isolation. If we end up continuing use of Roaster (e.g. to replace jsonschema2pojo), we can at least re-factor and put this sort of code in some utility class or some better place. IMO since it's only used for this one function here, it could stay as-is for now. wdyt?

Reply:
yup 👍